<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>KstMonochromeDialog</class>
  <widget class="QDialog" name="KstMonochromeDialog" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>576</width>
        <height>420</height>
      </rect>
    </property>
    <property name="windowTitle" >
      <string>Monochrome Print Configuration</string>
    </property>
    <layout class="QVBoxLayout" >
      <item>
        <layout class="QHBoxLayout" >
          <property name="margin" >
            <number>0</number>
          </property>
          <item>
            <widget class="QCheckBox" name="enhanceReadability" >
              <property name="text" >
                <string>Cycle curve properties</string>
              </property>
            </widget>
          </item>
          <item>
            <spacer name="spacer33" >
              <property name="sizeHint" >
                <size>
                  <width>240</width>
                  <height>20</height>
                </size>
              </property>
              <property name="sizeType" >
                <enum>Expanding</enum>
              </property>
              <property name="orientation" >
                <enum>Horizontal</enum>
              </property>
            </spacer>
          </item>
        </layout>
      </item>
      <item>
        <widget class="Q3ButtonGroup" name="cycleOrderGroup" >
          <property name="title" >
            <string>Property Selection and Cycle Order</string>
          </property>
          <layout class="QGridLayout" >
            <property name="margin" >
              <number>0</number>
            </property>
            <item rowspan="3" row="0" column="0" colspan="1" >
              <layout class="QVBoxLayout" >
                <property name="margin" >
                  <number>0</number>
                </property>
                <item>
                  <widget class="QLabel" name="textLabel1" >
                    <property name="text" >
                      <string>Available:</string>
                    </property>
                    <property name="wordWrap" >
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="Q3ListBox" name="availableListBox" >
                    <item>
                      <property name="text" >
                        <string>New Item</string>
                      </property>
                    </item>
                  </widget>
                </item>
              </layout>
            </item>
            <item row="1" column="1" >
              <layout class="QGridLayout" >
                <property name="margin" >
                  <number>0</number>
                </property>
                <item row="1" column="1" >
                  <widget class="QPushButton" name="_up" >
                    <property name="sizePolicy" >
                      <sizepolicy>
                        <hsizetype>0</hsizetype>
                        <vsizetype>0</vsizetype>
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                      </sizepolicy>
                    </property>
                    <property name="text" >
                      <string/>
                    </property>
                    <property name="shortcut" >
                      <string>Alt+Up</string>
                    </property>
                    <property name="toolTip" stdset="0" >
                      <string>Raise in cycle order: Alt+up</string>
                    </property>
                  </widget>
                </item>
                <item row="3" column="1" >
                  <widget class="QPushButton" name="_down" >
                    <property name="sizePolicy" >
                      <sizepolicy>
                        <hsizetype>0</hsizetype>
                        <vsizetype>0</vsizetype>
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                      </sizepolicy>
                    </property>
                    <property name="text" >
                      <string/>
                    </property>
                    <property name="shortcut" >
                      <string>Alt+Down</string>
                    </property>
                    <property name="toolTip" stdset="0" >
                      <string>Lower in cycle order: Alt+down</string>
                    </property>
                  </widget>
                </item>
                <item row="2" column="2" >
                  <widget class="QPushButton" name="_add" >
                    <property name="sizePolicy" >
                      <sizepolicy>
                        <hsizetype>0</hsizetype>
                        <vsizetype>0</vsizetype>
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                      </sizepolicy>
                    </property>
                    <property name="text" >
                      <string/>
                    </property>
                    <property name="toolTip" stdset="0" >
                      <string>Select: Alt+s</string>
                    </property>
                  </widget>
                </item>
                <item row="2" column="0" >
                  <widget class="QPushButton" name="_remove" >
                    <property name="sizePolicy" >
                      <sizepolicy>
                        <hsizetype>0</hsizetype>
                        <vsizetype>0</vsizetype>
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                      </sizepolicy>
                    </property>
                    <property name="text" >
                      <string/>
                    </property>
                    <property name="toolTip" stdset="0" >
                      <string>Remove: Alt+r</string>
                    </property>
                  </widget>
                </item>
              </layout>
            </item>
            <item rowspan="3" row="0" column="2" colspan="1" >
              <layout class="QVBoxLayout" >
                <property name="margin" >
                  <number>0</number>
                </property>
                <item>
                  <widget class="QLabel" name="textLabel2" >
                    <property name="text" >
                      <string>Selected:</string>
                    </property>
                    <property name="wordWrap" >
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="Q3ListBox" name="selectedListBox" >
                    <item>
                      <property name="text" >
                        <string>New Item</string>
                      </property>
                    </item>
                  </widget>
                </item>
              </layout>
            </item>
            <item row="0" column="1" >
              <spacer name="spacer42" >
                <property name="sizeHint" >
                  <size>
                    <width>20</width>
                    <height>31</height>
                  </size>
                </property>
                <property name="sizeType" >
                  <enum>Expanding</enum>
                </property>
                <property name="orientation" >
                  <enum>Vertical</enum>
                </property>
              </spacer>
            </item>
            <item row="2" column="1" >
              <spacer name="spacer42_3" >
                <property name="sizeHint" >
                  <size>
                    <width>20</width>
                    <height>40</height>
                  </size>
                </property>
                <property name="sizeType" >
                  <enum>Expanding</enum>
                </property>
                <property name="orientation" >
                  <enum>Vertical</enum>
                </property>
              </spacer>
            </item>
          </layout>
        </widget>
      </item>
      <item>
        <widget class="Q3ButtonGroup" name="optionsGroup" >
          <property name="title" >
            <string>Options</string>
          </property>
          <layout class="QGridLayout" >
            <property name="margin" >
              <number>0</number>
            </property>
            <item row="0" column="1" >
              <spacer name="spacer36" >
                <property name="sizeHint" >
                  <size>
                    <width>16</width>
                    <height>20</height>
                  </size>
                </property>
                <property name="sizeType" >
                  <enum>Expanding</enum>
                </property>
                <property name="orientation" >
                  <enum>Horizontal</enum>
                </property>
              </spacer>
            </item>
            <item row="0" column="0" >
              <layout class="QGridLayout" >
                <property name="margin" >
                  <number>0</number>
                </property>
                <item row="0" column="1" >
                  <widget class="QSpinBox" name="maxLineWidth" />
                </item>
                <item row="0" column="0" >
                  <widget class="QLabel" name="textLabel3" >
                    <property name="text" >
                      <string>Maximum line width:</string>
                    </property>
                    <property name="wordWrap" >
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
              </layout>
            </item>
            <item row="0" column="2" >
              <layout class="QGridLayout" >
                <property name="margin" >
                  <number>0</number>
                </property>
                <item row="0" column="0" >
                  <widget class="QLabel" name="textLabel4" >
                    <property name="text" >
                      <string>Point density:</string>
                    </property>
                    <property name="wordWrap" >
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
                <item row="0" column="1" >
                  <widget class="QComboBox" name="pointDensity" >
                    <item>
                      <property name="text" >
                        <string>All</string>
                      </property>
                    </item>
                    <item>
                      <property name="text" >
                        <string>High</string>
                      </property>
                    </item>
                    <item>
                      <property name="text" >
                        <string>Medium</string>
                      </property>
                    </item>
                    <item>
                      <property name="text" >
                        <string>Low</string>
                      </property>
                    </item>
                  </widget>
                </item>
              </layout>
            </item>
            <item row="0" column="3" >
              <spacer name="spacer37" >
                <property name="sizeHint" >
                  <size>
                    <width>40</width>
                    <height>20</height>
                  </size>
                </property>
                <property name="sizeType" >
                  <enum>Expanding</enum>
                </property>
                <property name="orientation" >
                  <enum>Horizontal</enum>
                </property>
              </spacer>
            </item>
          </layout>
        </widget>
      </item>
      <item>
        <layout class="QHBoxLayout" >
          <property name="margin" >
            <number>0</number>
          </property>
          <item>
            <spacer name="spacer35" >
              <property name="sizeHint" >
                <size>
                  <width>300</width>
                  <height>20</height>
                </size>
              </property>
              <property name="sizeType" >
                <enum>Expanding</enum>
              </property>
              <property name="orientation" >
                <enum>Horizontal</enum>
              </property>
            </spacer>
          </item>
          <item>
            <widget class="QPushButton" name="_Cancel" >
              <property name="text" >
                <string>&amp;Close</string>
              </property>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
  <tabstops>
    <tabstop>enhanceReadability</tabstop>
    <tabstop>selectedListBox</tabstop>
    <tabstop>availableListBox</tabstop>
    <tabstop>maxLineWidth</tabstop>
    <tabstop>pointDensity</tabstop>
    <tabstop>_Cancel</tabstop>
  </tabstops>
</ui>
